main: Add '--version' CLI flag

har7an opened the pull request:
Prints the current version as defined in the `Cargo.toml`.

kelnos commented on espmonitor/src/main.rs:
```diff
@@ -43,6 +43,9 @@ fn parse_args() -> Result<Option<AppArgs>, Box<dyn Error>> {
     if args.contains("-h") || args.contains("--help") {
         print_usage();
         Ok(None)
+    } else if args.contains("-v") || args.contains("--version") {
```
GNU standard is generally a _capital_ `V` for the short option.  (Lowercase `v` is used for `--verbose`.)

kelnos commented:
Thanks for this!  Probably should just move to a more full-featured command line parser like `clap` that does all this for us, but this is great for now.  Just one small change and I'm happy to merge.

kelnos commented:
Oh -- it looks like the cargo integration doesn't do this by itself, so if you'd be interested in adding it there as well, that'd be great.

har7an commented:
Thanks for the feedback. I added the requested change and copied the same code to the cargo-esptools. I also updated the usage message to reflect the new `--version` option.

har7an commented:
> Probably should just move to a more full-featured command line parser like `clap` that does all this for us

I've got some experience with clap. Shall I go ahead and try to add this?

kelnos commented:
Ok, we're working on an impl with `clap` now, so I guess we can close this out.
